frontend/src/app/components/channel/large/large.controller.ts

```diff
@@ -0,0 +1,13 @@
+import { BaseChannelController } from '../channel.controller';
+
+/** @ngInject */
+export class LargeChannelController extends BaseChannelController {
+  next() {
```

Comment:
Consider moving next to Base. So that other channels (for controller, fours will have different next) will have next capability.